app/modules/booking/endpoints_booking.py

```diff
@@ -265,26 +269,45 @@ async def create_booking(
     )
     await cruds_booking.create_booking(booking=db_booking, db=db)
     result = await cruds_booking.get_booking_by_id(db=db, booking_id=db_booking.id)
-
     try:
-        if result:
-            now = datetime.now(UTC)
+        manager_group_id = result.room.manager_id
+        manager_group = await cruds_groups.get_group_by_id(
+            db=db,
+            group_id=manager_group_id,
+        )
+
+        applicant_user = await cruds_users.get_user_by_id(
+            db=db,
+            user_id=result.applicant_id,
+        )
```

Comment:
I think the applicant should always be the user making the request

Reply:
I don't really understant what you mean here.